improver/cli/spot_extract.py

```diff
@@ -39,13 +39,14 @@
 def process(
     *cubes: cli.inputcube,
     apply_lapse_rate_correction=False,
```

Comment:
You've changed all the others...
```suggestion
    apply_lapse_rate_correction: bool = False,
```